docs/public/openapi-security.json

```
@@ -0,0 +1,187 @@
+                "properties": {
+                  "prop1": {
+                    "type": "string"
+                  },
+                  "prop2": {
+                    "type": "number"
+                  },
+                  "prop3": {
+                    "type": "array",
+                    "items": {
+                      "type": "string",
+                      "example": "value1",
```

Biome (1.9.4)

[error] The key example was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)